dora/services/migrations/0105_update_orientation_modes.py

```diff
@@ -0,0 +1,126 @@
+from django.conf import settings
+from django.db import migrations
+from django.db.models import Q
+
+from dora.services.enums import ServiceStatus
+
+
+def is_orientable(service):
+    structure_blacklisted = False
+    for siren in settings.ORIENTATION_SIRENE_BLACKLIST:
+        if service.structure.siret and service.structure.siret.startswith(siren):
+            structure_blacklisted = True
+            break
```

Comment:
_nit_ : il ya peut être un poil plus court qu'itérer sur la liste (`in` sur compréhension ?).

Reply:
Je ne pourrais pas break dans une compréhension de liste. À quoi penses-tu exactement ?